hub-js/src/local/storage/service.ts

```diff
@@ -57,14 +94,17 @@ export default class DelegatedStorageService {
    * @param inputs - Describes what should be stored.
    * @returns The update backend's context. If there was no update, it's undefined.
    *
+   *  TODO(https://github.com/capactio/capact/issues/604): validate if `input.value` is allowed by backend (`backend.acceptValue`)
+   *  TODO(https://github.com/capactio/capact/issues/604): validate `input.backend.context` against `backend.contextSchema`.
```

Comment:
Nope, it's not a scope of #634. It was a stretch for #645, but it will be extracted to dedicated task as we won't be able to finish it in a given estimation.